build.proj

```diff
@@ -33,8 +33,10 @@
     <ExeExtension>.exe</ExeExtension>
     <ExeExtension Condition=" '$(OS)' != 'Windows_NT' "></ExeExtension>
     <DotnetStage0>$(Stage0Path)/dotnet$(ExeExtension)</DotnetStage0>
-
     <DotnetCliBuildDirectory>$(RepoRoot)/build_projects/dotnet-cli-build</DotnetCliBuildDirectory>
+    <OutputDirectory>$(RepoRoot)/artifacts</OutputDirectory>
+    <PackagesDir>$(RepoRoot)/.nuget</PackagesDir>
```

Comment:
This name is confusing compared to PackagesDirectory. We need to better disambiguate.